src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -10,6 +10,7 @@
 import teammates.common.datatransfer.InstructorAttributes;
```

Comment:
Ideally, the account should be deleted if it doesn't have any other student
or instructor objects. But no big deal if it is not deleted.